migrator/cf.py

```diff
@@ -29,3 +31,7 @@ def get_space_id_for_service_instance_id(instance_id, client):
 def get_org_id_for_space_id(space_id, client):
     response = client.v3.spaces.get(space_id)
     return response["relationships"]["organization"]["data"]["guid"]
+
+def get_service_plan_visibility_id_for_org_id(org_id, client):
+    response = client.v3.organizations.get(org_id)
+    return response["relationships"][""]
```

Comment:
This looks incomplete...

Reply:
D'oh!  This was leftover from my early attempts at getting the right information and I forgot to take that out.

I don't think it's needed because when enabling a service plan, that new service plan visibility object is created, which is unique to the plan and the org; disabling a service plan just requires the GUID for the service plan visibility object, no org information.